src/angular-business/header/header/header.component.scss

```diff
@@ -74,31 +101,35 @@ $sbbHeaderHeight: 54px;
   display: flex;
   flex-grow: 0;
   flex-direction: column;
-  width: 200px;
-  margin-left: 15px;
+  width: pxToRem(200);
+  margin-left: pxToRem(15);
   font-family: $fontSbbLight;
 
+  :host(.sbb-header-flexible) & {
+    width: auto;
+    margin-right: 2rem;
```

Comment:
should we not use UX predefined dividers?

Reply:
Changed to 2.5rem